pkg/exporter/elasticsearch/factory.go

```diff
@@ -12,19 +12,19 @@ import (
 
 const (
 	typeStr = "jaeger_elasticsearch"
```

Comment:
Although not part of this PR, just wondering whether (as this exporter is located in the Jaeger repo) whether we need the `jaeger_` prefix?

And at some point, when this component is perhaps not so Jaeger specific - and operating off the OpenTelemetry proto model, it might be a candidate for moving to the contrib repo.

Reply:
That would make sense if the module was storing OTEL model and there was also supported OT query API.

Reply:
On the `jaeger` prefix: 

I think it makes sense to distinguish Jaeger components. At the moment there is `jaeger` receiver, `jaeger_grpc` and `jaeger_thrift_http` exporters. For the consistency reasons, I would keep the prefix. If there will be plans in the future to support storage layer in OTEL community those probably will not have vendor prefix.